src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div id="business-type-confirm">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label class="d-block title-label">Business Type</label>
+        <v-chip
+          v-if="hasBusinessTypeChecked"
+          id="checked-chip"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          Checked
+        </v-chip>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <p class="mb-0">General Partnership</p>
+        <div id="business-check-div">
+          <v-checkbox
+            class="mt-0"
+            v-model="checked"
+            hide-details
+            label="I acknowledge that a General Partnership cannot be changed into a
+            Sole Proprietorship (including DBA). If this is necessary, a new
+            Name Request Number and Statement of Registration (along with
+            associated fees) will be required."
+          >
+          </v-checkbox>
+        </div>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+
+@Component
+export default class BusinessTypeConfirm extends Vue {
+  /** The registration business number. */
+  @Prop({ required: true })
+  readonly businessTypeConfirm!: boolean
+
+  /** Whether to display Change features. */
+  @Prop({ default: false })
+  readonly hasBusinessTypeChecked!: boolean;
+
+  // Local variables
+  protected checked = false;
+
+  /** Called when component is mounted. */
+  protected mounted (): void {
+    // init model variable + validate
+    this.checked = this.businessTypeConfirm
+    this.emitValid()
+  }
+
+  /** Watches for changes to the checkbox value. */
+  @Watch('checked')
+  private onValueChanged (): void {
+    // update business type + validate
+    this.emitBusinessType()
+    this.emitValid()
+  }
+
+  /** Emits updated business type event. */
+  @Emit('update:businessTypeConfirm')
+  private emitBusinessType (): boolean {
+    return this.checked
+  }
+
+  // /** Emits component validity event. */
+  @Emit('valid')
+  private emitValid (): boolean {
+    return this.checked
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "@/assets/styles/theme.scss";
+
+#business-type-confirm {
+  .col-sm-3 {
+    color: $gray9;
+    font-weight: bold;
+  }
+
+  .col-sm-9 {
+    color: $gray7;
+  }
+}
+
+#business-check-div {
+  display: flex;
+  margin-top: 20px;
+}
+
+#business-check-div .mt-0 {
+  margin-left: -3px;
+  padding-top: 1px;
+}
+
+::v-deep .v-input--checkbox .v-input__slot {
```

Comment:
This was added in the commit: ac78c6c... to align the checkbox and the label.

Reply:
It's nice to have a code comment here to say what you're doing.